drivers/clock_control/clock_control_mcux_scg.c

```diff
@@ -60,33 +61,56 @@ static int mcux_scg_get_rate(const struct device *dev,
 	case KINETIS_SCG_FIRC_CLK:
 		clock_name = kCLOCK_ScgFircClk;
 		break;
+#if (FSL_FEATURE_SCG_HAS_SPLL)
 	case KINETIS_SCG_SPLL_CLK:
 		clock_name = kCLOCK_ScgSysPllClk;
 		break;
+#endif
+#if (FSL_FEATURE_SCG_HAS_LPFLL)
+	case KINETIS_SCG_SPLL_CLK:
+		clock_name = kCLOCK_ScgLpFllClk;
+		break;
+#endif
+#if (FSL_FEATURE_SCG_HAS_SOSCDIV1)
```

Comment:
```suggestion
#if (defined(FSL_FEATURE_SCG_HAS_SOSCDIV1) && FSL_FEATURE_SCG_HAS_SOSCDIV1)
```

Please follow the same pattern for the remaining `#if` statements.